src/core/polyhedral/codegen_llvm.cc

```diff
@@ -195,8 +210,46 @@ class IslAstExprInterpeter {
   }
 };
 
-DEFINE_bool(llvm_dump_before_opt, false, "Print IR before optimization");
-DEFINE_bool(llvm_dump_after_opt, false, "Print IR after optimization");
+namespace {
+// In a separate process, call the LLC and dump the ASM
+// Since this is for debugging purposes only atm, separate process is fine.
+string emitAsm(const string& llvmIr) {
+  // Create a temporary file (do not use racy tmpname;
```

Comment:
This seems hacky, I'm fairly confident LLVM has a nice way to emit assembly...

Reply:
I looked into the LLVM / LLC doc and got a nice refresher on machine instructions instruction selection etc. Still I couldn't find a way to easily plugin the code you shared with me. Care to add the option yourself in a separate PR? It'll take you significantly less time than it took me, and I can just scrap this :)
Another possibility is to land this and have you do a followup cleanup PR; calling a separate LLC process is done only in the debug mode and it's not like we don't have similar external ```popen``` behavior somewhere in the codebase ;)

Your call, just lmk what you prefer, thanks!

Reply:
I'd prefer removing this for now, then adding in later with the proper way unless its absolutely necessary right now simply because of the messiness we already have from popen/path-based things (aka ldconfig library lookup).